src/stories/containers/FinancesOverview/components/CostBreakdownTable/ByBudgetTableRow.tsx

```diff
@@ -156,81 +183,97 @@ const NameColumn = styled.div({
   marginBottom: 9.2,
   lineHeight: '14px',
   fontSize: 14,
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
 
-  [lightTheme.breakpoints.up('table_834')]: {
+  [lightTheme.breakpoints.up('tablet_768')]: {
     paddingLeft: 16,
     paddingRight: 4,
     marginBottom: 0,
   },
 
-  [lightTheme.breakpoints.up('desktop_1194')]: {
+  [lightTheme.breakpoints.up('desktop_1024')]: {
     lineHeight: '23px',
   },
+
+  [lightTheme.breakpoints.up('desktop_1280')]: {
+    whiteSpace: 'normal',
+    paddingRight: 0,
+  },
 });
 
 const TotalPercentageColumn = styled.div({
-  width: 145,
-  minWidth: 145,
+  flex: 1.1,
+  marginLeft: 'auto',
 
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
-    width: 183.5,
-    minWidth: 183.5,
+  [lightTheme.breakpoints.up('tablet_768')]: {
+    flex: 1,
   },
 
-  [lightTheme.breakpoints.up('desktop_1194')]: {
-    width: 170,
-    minWidth: 170,
+  [lightTheme.breakpoints.up('desktop_1280')]: {
+    width: 166,
+    minWidth: 166,
   },
 
   [lightTheme.breakpoints.up('desktop_1440')]: {
-    width: 180,
-    minWidth: 180,
+    width: 184,
+    minWidth: 184,
   },
 });
 
 const TotalSpendColumn = styled.div({
   width: 153,
   minWidth: 153,
 
-  [lightTheme.breakpoints.up('table_834')]: {
+  [lightTheme.breakpoints.up('tablet_768')]: {
     textAlign: 'right',
+    flex: 0.85,
+    minWidth: 'auto',
+    width: 'auto',
   },
 
-  [lightTheme.breakpoints.between('table_834', 'desktop_1194')]: {
-    width: 183.5,
-    minWidth: 183.5,
+  [lightTheme.breakpoints.between('tablet_768', 'desktop_1024')]: {
     paddingRight: 4,
   },
 
+  [lightTheme.breakpoints.up('desktop_1024')]: {
+    flex: 0.95,
+  },
+
   [lightTheme.breakpoints.up('desktop_1280')]: {
-    width: 157,
-    minWidth: 157,
+    width: 140,
+    minWidth: 140,
+  },
+
+  [lightTheme.breakpoints.up('desktop_1440')]: {
+    width: 150,
+    minWidth: 150,
   },
 });
 
 const ViewColumn = styled.div({
-  width: 70,
-  minWidth: 70,
   textAlign: 'center',
-  display: 'none',
+  display: 'block',
+  borderLeft: `1px solid ${'#D4D9E1'}`,
```

Comment:
Can you add direct the values of color, instead of use template literals '1px solid #D4D9E1'